contexts/cloud-messaging.provider.tsx

```
@@ -0,0 +1,75 @@
+  useContext,
+  createContext,
+  type PropsWithChildren,
+  useEffect,
+} from "react";
+import messaging from "@react-native-firebase/messaging";
+import { Alert } from "react-native";
+
+import { PermissionsAndroid } from 'react-native';
+PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.POST_NOTIFICATIONS);
+
+interface CloudMessagingContextProps { }
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)